build.sbt

```diff
@@ -214,6 +214,7 @@ lazy val docs = http4sProject("docs")
   .settings(ghpages.settings)
   .settings(tutSettings)
   .settings(
+    libraryDependencies <+= scalazVersion {szv => argonautShapeless(szv) },
```

Comment:
I'm personally using argonaut because of the ability to change the sum codec derivation. I switched over from circe. Let's switch in that case.